src/aliceVision/hdr/sampling.cpp

```diff
@@ -202,9 +202,9 @@ bool Sampling::extractSamplesFromImages(std::vector<ImageSample>& out_samples, c
             integral(imgIntegral, blockInput);
             integral(imgIntegralSquare, imgSquare);
 
-            for (int i = params.radius + 1; i < imgIntegral.Height() - params.radius; ++i)
+            for(int i = radiusp1; i < imgIntegral.Height() - params.radius; ++i)
```

Comment:
maybe reduce the scope of `radiusp1` and declare it here before the `for`

Reply:
It's a constant based on the input params, so I think it's better to keep it out of the loop with all the variations around the input params.